scripts/qa_crowbarsetup.sh

```diff
@@ -1471,6 +1471,10 @@ EOF
         if ! grep -q /var/lib/glance/images $f; then
             echo "/var/lib/glance/images     <%= @admin_subnet %>/<%= @admin_netmask %>(rw,async,no_root_squash,no_subtree_check)" >> $f
         fi
+        mkdir -p /srv/nfs/{database,rabbitmq}
+        if ! grep -q /srv/nfs $f; then
+            echo "/srv/nfs     <%= @admin_subnet %>/<%= @admin_netmask %>(rw,async,no_root_squash,no_subtree_check)" >> $f
```

Comment:
Because it's one mount resource per service in pacemaker. Otherwise, rabbitmq and postgresql have to be on the same node.